src/SFA.DAS.Commitments.Application/Commands/ApproveTransferRequest/ApproveTransferRequestCommandHandler.cs

```diff
@@ -69,22 +66,13 @@ protected override async Task HandleCore(ApproveTransferRequestCommand command)
 
             _historyService.TrackUpdate(commitment, CommitmentChangeType.TransferSenderApproval.ToString(), commitment.Id, null, CallerType.TransferSender, command.UserEmail, commitment.ProviderId, command.TransferSenderId, command.UserName);
 
-            if (_featureToggleService.IsEnabled("ManageReservations"))
-            {
+            
```

Comment:
The message written to the log should be updated to remove reference to the (now) non-existent feature. The message itself is not much use from a diagnosis pov - perhaps we could add the employer account id to it?